packages/@ama-sdk/core/src/plugins/index.ts

```diff
@@ -24,6 +23,7 @@ export * from './retry/index';
 export * from './reviver/index';
 export * from './session-id/index';
 export * from './si-token/index';
+export * from './simple-api-key-authentication/index';
```

Comment:
You will need to add this new plugin into the listed plugins in then readme.md of the package